plugins/inputs/memcached/README.md

```diff
@@ -0,0 +1,95 @@
+# Memcached Input Plugin
+
+This plugin gathers statistics data from a Memcached server.
+
+* get_hits
+* get_misses
+* evictions
+* limit_maxbytes
+* bytes
+* uptime
+* curr_items
+* total_items
+* curr_connections
+* total_connections
+* connection_structures
+* cmd_get
+* cmd_set
+* delete_hits
+* delete_misses
+* incr_hits
+* incr_misses
+* decr_hits
+* decr_misses
+* cas_hits
+* cas_misses
+* bytes_read
+* bytes_written
+* threads
+* conn_yields
+
+### Configuration:
+
+```toml
+# Read metrics from one or many memcached servers.
+[[inputs.memcached]]
+	# An array of address to gather stats about. Specify an ip on hostname
+	# with optional port. ie localhost, 10.0.0.1:11211, etc.
+	servers = ["localhost:11211"]
+	# An array of unix memcached sockets to gather stats about.
+	# unix_sockets = ["/var/run/memcached.sock"]
+```
+
+### Measurements & Fields:
+
+The fields from this plugin are gathered in the *memcached* measurement.
+
+Fields:
+
+* get_hits - Number of keys that have been requested and found present
+* get_misses - Number of items that have been requested and not found
+* evictions - Number of valid items removed from cache to free memory for new items
+* limit_maxbytes - Number of bytes this server is allowed to use for storage
+* bytes - Current number of bytes used to store items
+* uptime - Number of secs since the server started
+* curr_items - Current number of items stored
+* total_items - Total number of items stored since the server started
+* curr_connections - Number of open connections
+* total_connections - Total number of connections opened since the server started running
+* connection_structures - Number of connection structures allocated by the server
+* cmd_get - Cumulative number of retrieval reqs
+* cmd_set - Cumulative number of storage reqs
+* delete_hits - Number of deletion reqs resulting in an item being removed
+* delete_misses - umber of deletions reqs for missing keys
+* incr_hits - Number of successful incr reqs
+* incr_misses - Number of incr reqs against missing keys
+* decr_hits - Number of successful decr reqs
+* decr_misses - Number of decr reqs against missing keys
+* cas_hits - Number of successful CAS reqs
+* cas_misses - Number of CAS reqs against missing keys
+* bytes_read - Total number of bytes read by this server from network
+* bytes_written - Total number of bytes sent by this server to network
+* threads - Number of worker threads requested
+* conn_yields - Number of times any connection yielded to another due to hitting the -R limit
+
+Description of gathered fields taken from [here](https://github.com/memcached/memcached/blob/master/doc/protocol.txt).
+
+### Tags:
+
+* Memcahed measurements have the following tags:
+    - server (the host name from which metrics are gathered)
+
+### Sample Queries:
+
+These are some useful queries (to generate dashboards or other) to run against data from this plugin:
```

Comment:
Instead of this boilerplate, maybe you can describe what at a high level what the query will return.